rero/rero-ils-ui
# admin & public-search: add missing translations

* Adds missing translations.

Co-Authored-by: Alicia Zangger <alicia.zangger@rero.ch>

## Why are you opening this PR?

Part of https://github.com/rero/rero-ils-ui/issues/43
Closes points 10, 18, 20 and 31.

## How to test?

Check menu and editor translations in admin app.
Check facets translations in public search app.

## Code review check list
